go/encoding/basex/encoding.go

```diff
@@ -86,6 +94,21 @@ func (enc *Encoding) Encode(dst, src []byte) {
 	}
 }
 
+// extraBits returns the number of bits in the encoding that we'll need
+// to shift over.
+func (enc *Encoding) extraBits(decodedLen, encodedLen int) uint {
+	var encodedBits int
+	if encodedLen == enc.outBlockLen {
+		encodedBits = enc.maxEncodedBitsPerBlock
+	} else {
+		encodedBits = int(math.Floor(enc.logOfBase * float64(encodedLen)))
```

Comment:
Am I reading it right, that this branch only happens for short blocks? Could we comment that?